src/main.py

```diff
@@ -1,13 +1,28 @@
 
 import pytorch_lightning as pl
+from torch import optim
 from pytorch_lightning.loggers import CSVLogger
 from ModelInterface import ModelInterface
 from datamodel.CIFAR10DataModel import CIFAR10DataModule
 from pytorch_lightning.callbacks import Callback
 import time
 
+
 if __name__ == '__main__':
 
+    logger_dir = "logs"
+    logger_name = "resnet32_cifar"
+    max_epochs = 5
+    gpus = 0
+    max_time = "00:01:00:00"
+    check_val_every_n_epoch = 1
+    log_every_n_steps = 1
+    strategy = "ddp"
+    optimizer = optim.SGD
+    lr = 1e-2
+    model_name = "ResNet32"
+
+    lr_monitor = pl.callbacks.LearningRateMonitor(logging_interval='step')
```

Comment:
I used variables instead. I have no idea whether it's the right way to modify the hardcode.